lib/kits/core-ui/components/common/shopping-list.ts

```diff
@@ -113,13 +114,15 @@ const defaultTemplate = `\
     </div>\
 `;
 
-const ShoppingList = ({template}) => {
+const ShoppingList = ({template, scriptEls, sgnData}) => {
```

Comment:
Please put the actual types, these are already well-known: 

https://github.com/tjek/tjek-js-sdk/blob/develop/lib/kits/core-ui/paged-publication.ts#L41 

https://github.com/tjek/tjek-js-sdk/blob/develop/lib/kits/core-ui/components/paged-publication/main-container.ts#L60